utils/helpers/get-company-page-title.ts

```diff
@@ -0,0 +1,19 @@
+import {
+  isAssociation,
+  isServicePublic,
+  isCollectiviteTerritoriale,
+  IUniteLegale,
+} from '#models/index';
+
+export const getCompanyPageTitle = (uniteLegale: IUniteLegale) => {
+  switch (true) {
```

Comment:
I feel like `- SIREN ${uniteLegale.siren}, ${uniteLegale.siege.siret}, ${uniteLegale.libelleActivitePrincipale} | Annuaire Entreprises` is repeated on every line !

```suggestion
const description = `- SIREN ${uniteLegale.siren}, ${uniteLegale.siege.siret}, ${uniteLegale.libelleActivitePrincipale} | Annuaire Entreprises`;

case isAssociation(uniteLegale):
    return `Association ${description}`;
 case isServicePublic(uniteLegale):
      return `Service public ${description}`
      
  ...
```